pkg/contracts/src/RegistryFactoryV0_0.sol

```diff
@@ -47,17 +47,20 @@ contract RegistryFactoryV0_0 is OwnableUpgradeable, UUPSUpgradeable {
     }
 
     function initialize(
+        address _owner,
         address _gardensFeeReceiver,
         address _registryCommunityTemplate,
+        address _strategyTemplate,
         address _collateralVaultTemplate
     ) public virtual initializer {
-        __Ownable_init();
+        ProxyOwnableUpgrader.initialize(_owner);
```

Comment:
make sure initialize call ownable_init

Reply:
Implication of that will be OwnerShipTransfered event will be emitted twice 0x -> sender -> specified owner

Reply:
Oh yeah just realized I can pass this owner directly to the init function

Reply:
https://github.com/1Hive/gardens-v2/pull/361#discussion_r1726171409